testapp/src/main/res/layout/settings_menu_activity.xml

```diff
@@ -189,11 +189,11 @@
             android:background="@color/bg_section" />
 
         <com.google.android.material.switchmaterial.SwitchMaterial
-            android:id="@+id/switchTestMode"
+            android:id="@+id/switchPreviewMode"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginTop="@dimen/small_12"
-            android:text="@string/lb_test_mode"
+            android:text="@string/lb_preview_mode"
             android:visibility="gone" />
```

Comment:
We can make this switch visible now.

Also, I think it needs to be updated to match the styling of the buttons. Maybe it just needs to be wrapped in `RelativeLayout` and add the horizontal divider beneath?

Reply:
Yeah, the Preview switch is visible now.